### DISCUSSION TITLE
train the model

### DISCUSSION THREAD

**Discussion by deepaks11** [ultralytics/ultralytics#4287] [General]
1. Can I train the facial emotion dataset using YOLOv8?
2. Can I train a model to recognize different types of faces along with their names and identify the faces in real-time?

**Comment by glenn-jocher** [comment]
@deepaks11 1. Yes, you can train YOLOv8 on a facial emotion dataset. YOLOv8 is versatile and can be used for various computer vision tasks, including the detection of facial emotions. You'll need to prepare your dataset with labeled emotions on faces for training.

2. YOLOv8 can also be trained to recognize different types of faces along with their names. This would involve training the model on a dataset with labeled faces. Once trained, YOLOv8 can identify these faces in real-time. Keep in mind that for privacy and ethical considerations, you should have the necessary permissions to use and process facial data.

For detailed instructions on training and real-time inference, please refer to our documentation at https://docs.ultralytics.com. 😊
